source/features/insert-summary-button.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import * as icons from '../libs/icons';
+
+const init = () => {
+  const addSummaryDetails = () => {
+    const commentField = select<HTMLTextAreaElement>('.js-comment-field');
+
+    if (!commentField) {
+      return false;
+    }
```

Comment:
No need to `select` and to check for its existence; `addSummaryDetails` is called as an event handler so you can use `const commentField = event.target`

Reply:
I couldn't figure out how to just get `event.target` to work since the target was technically the button itself, and it was saving the svg `path` to the variable 🤔 So I took a similar approach and used the `event.target` to find the closest `form`, then grabbed the `textarea` of said form. Hopefully this is what you were hinting at!

Reply:
Duh, my bad. I thought that the event was being fired from `textarea`